pkg/apis/kops/v1alpha1/componentconfig.go

```diff
@@ -330,10 +330,8 @@ type KubeSchedulerConfig struct {
 	Image string `json:"image,omitempty"`
 	// LeaderElection defines the configuration of leader election client.
 	LeaderElection *LeaderElectionConfiguration `json:"leaderElection,omitempty"`
-	// PolicyConfigMap is the name of configmap to use for scheduler policy
-	PolicyConfigMap string `json:"policyConfigMap,omitempty" flag:"policy-configmap"`
-	// PolicyConfigMapNamespace is the namespace containing the configmap
-	PolicyConfigMapNamespace string `json:"policyConfigMapNamespace,omitempty" flag:"policy-configmap-namespace"`
```

Comment:
Is there any reason that this is not kube-system?  From a best practice standpoint would we ever put is somewhere else?

Reply:
The default value of --policy-configmap-namespace is already kube-system, so I removed the configuration option